packages/google_maps_flutter/google_maps_flutter/ios/Classes/JsonConversions.m

```diff
@@ -68,4 +68,43 @@ + (UIColor *)toColor:(NSNumber *)numberColor {
   return holes;
 }
 
++ (GMUWeightedLatLng *)toWeightedLatLng:(NSArray *)data {
+  return
+      [[GMUWeightedLatLng alloc] initWithCoordinate:[FLTGoogleMapJsonConversions toLocation:data[0]]
+                                          intensity:[FLTGoogleMapJsonConversions toDouble:data[1]]];
+}
+
++ (NSArray<GMUWeightedLatLng *> *)toWeightedData:(NSArray *)data {
+  NSMutableArray<GMUWeightedLatLng *> *weightedData = [[NSMutableArray alloc] init];
+  for (unsigned i = 0; i < [data count]; i++) {
+    GMUWeightedLatLng *weightedLatLng = [FLTGoogleMapJsonConversions toWeightedLatLng:data[i]];
+    [weightedData addObject:weightedLatLng];
+  }
+
+  return weightedData;
+}
+
++ (GMUGradient *)toGradient:(NSArray *)data {
+  NSMutableArray<UIColor *> *colors = [[NSMutableArray alloc] init];
+  NSMutableArray<NSNumber *> *startPoints = [[NSMutableArray alloc] init];
+  // Starting at 0 causes rendering issues
```

Comment:
Is this issue documented some where? For example, what is the issue we are preventing; why we use the value that we choose to use here; is there a plan to fix this workaround?

Reply:
This is from my personal experience in testing. I can document the issue more thoroughly if desired. I didn't think too hard about it because we might want to allow developers to specify their own startPoints.

Reply:
Sounds good! Let's let developer to specify their own startPoints.

Reply:
Done